packages/common/test/util/index.test.js

```diff
@@ -36,6 +36,11 @@ describe('encode', () => {
     expect(() => encode({})).to.throw(errorMsg);
     expect(() => encode(() => {})).to.throw(errorMsg);
   });
+  it('should encode a javascript object', () => {
```

Comment:
We might want to expand the tests in light of the extra complexity I've added above